modules/openapi-generator/src/test/resources/3_0/examples.yaml

```diff
@@ -118,3 +118,16 @@ components:
           properties:
             city:
               type: string
+    DummyUser:
+      type: object
+      properties:
+        name:
+          type: string
+          pattern: '^[a-z\s]+$'
+        number:
+          type: string
+          pattern: '^[0-9]{10}$'
+        address:
+          type: string
+          pattern: '^[a-z0-9\s]+$'
```

Comment:
How about updating these examples to include: leading / trailing / and /g  /i /m ?
That way we show that this feature is working and if it is changed in the future the test will break.
- slash use case `/^[a-z\s]+$/`
- /g use case `/^[a-z\s]+$/g`
- /i use case `/^[a-z\s]+$/i`
- /m use case `/^[a-z\s]+$/m`